libs/util/code-gen-types/src/dotnet-plugins.types.ts

```diff
@@ -127,6 +127,7 @@ export enum DotnetEventNames {
   CreateServerSecretsManager = "CreateServerSecretsManager",
   CreateEntityInterface = "CreateEntityInterface",
   CreateEntityExtensions = "CreateEntityExtensions",
+  CreateEntityModel = "CreateEntityModel",
```

Comment:
where is the plugin event internal function for this event?

Reply:
@overbit  Will be implemented on a separated PR from the Dotnet DSG

Reply:
Of course. Silly me